dask/dataframe/_compat.py

```diff
@@ -9,11 +9,7 @@
 from packaging.version import Version
 
 PANDAS_VERSION = Version(pd.__version__)
-PANDAS_GE_131 = PANDAS_VERSION >= Version("1.3.1")
-PANDAS_GE_133 = PANDAS_VERSION >= Version("1.3.3")
-PANDAS_GE_140 = PANDAS_VERSION >= Version("1.4.0")
-PANDAS_GE_150 = PANDAS_VERSION >= Version("1.5.0")
-PANDAS_GE_200 = PANDAS_VERSION.major >= 2
+PANDAS_GE_200 = True
```

Comment:
Non-blocking: I'm not a huge fan of setting this to `True` without ever verifying that the correct version of `pandas` is indeed installed. I think it would be nice if we confirmed that during `import dask.dataframe` and raised an `ImportError` if it's not the case. IIUC, it's entirely possible for a user to install "plain" Dask together with some old `pandas` version and attempt to use `dask.dataframe`, then having a bad time.

Consider this food for thought/non-blocking because we don't seem to have done this before as well.